cli/server/server.go

```diff
@@ -436,13 +436,21 @@ func mkP2PNotary(config config.P2PNotary, chain *core.Blockchain, serv *network.
 }
 
 func startServer(ctx *cli.Context) error {
+	var (
+		cfg config.Config
+		err error
+	)
+
 	if err := cmdargs.EnsureNone(ctx); err != nil {
 		return err
 	}
 
-	cfg, err := options.GetConfigFromContext(ctx)
-	if err != nil {
-		return cli.NewExitError(err, 1)
+	cfg = options.DefaultConfig
```

Comment:
Can't be, the behaviour must not be changed, it should use privnet config by default. Let's address the previous conversation first, and then deal with the others.